sqlx-core/src/logger.rs

```diff
@@ -23,44 +28,49 @@ impl<'q> QueryLogger<'q> {
     }
 
     pub(crate) fn finish(&self) {
-        let elapsed = self.start.elapsed();
-
-        let lvl = if elapsed >= SLOW_QUERY_THRESHOLD {
-            Level::Warn
+        let lvl = if let Some(start) = self.start {
+            let elapsed = start.elapsed();
```

Comment:
Looks like this assignment needs to move back outside the scope of the if branch here.

Reply:
Yes it appears my last commit was not good.